docker/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+concourse-db:
```

Comment:
I did read the comments in pivotal. As far as I understand, we want to automate the deployment of paas-bootstrap. If concourse or postgres docker images are updated between dev and prod, we may end up with different versions in different accounts.